docs/docs/migrate-data.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Migrating to v7: Migrating Data"
+---
+
+## Before you start
+
+Before you start the data migration process, make sure you have the following:
+
+- a tenant ID for the new v7 coral instance you have created (you can find this by inspecting the record in the `tenants` collection in your new mongo instance)
+- a site ID for the new v7 coral instance (inspect the one record in the `sites` collection)
+- a mongo instance with a replica of your v4 production data set
+- a local machine or virtual machine with sufficient RAM and processing power to run `mongoexport` and `mongoimport` on your data set. In our experience, large data sets (10+ GB) will require a minimum of 16GB RAM and 8 cores.
+- _before_ you start the migration process, it is a good idea to create several test users (one commenter, one moderator, one admin). Following the data migration, attempt to log in and comment/moderate as these users to verify that account and login related data was migrated correctly, and that authentication has bee configured correctly.
+
+## 1. Obtain JSON files for v4 data
+
+The first step in the data migration process is to get one JSON file for each collection in your v4 data. You will need to use [mongoexport](https://www.mongodb.com/docs/database-tools/mongoexport/) to get these files. The following bash script will create a folder containing the JSON files for each collection. This script may take a long time to run on large data sets.
+
+```bash
+# Set this to the TALK_MONGO_URL used by Coral.
```

Comment:
I'm sure this was copied from the README, however, it might be helpful to comment this as the "v4 TALK_MONGO_URL" so folks know we're exporting the v4 JSON out during this step and not the v7 instance. I know, this sounds very pedantic, I'm just trying to be almost ridiculously thorough on these steps.